server/container/guice/jpa-guice/src/main/resources/defaultMailetContainer.xml

```diff
@@ -49,7 +49,6 @@
         <mailet match="SMTPAuthSuccessful" class="RemoteDelivery">
             <outgoingQueue>outgoing</outgoingQueue>
             <delayTime>5000, 100000, 500000</delayTime>
-            <maxRetries>25</maxRetries>
```

Comment:
not `3` like in most configuration files?

Reply:
Not mentioning it achieve the same.

Reply:
and so, any reason to not be coherent between our configuration files?